collections/nemo_asr/nemo_asr/parts/manifest.py

```diff
@@ -157,18 +161,18 @@ def normalize_text(self, text, labels):
 
         # Turn all other punctuation to whitespace
         table = str.maketrans(punctuation, " " * len(punctuation))
-        text = self.english_string_normalization(
+        text = ManifestEN.english_string_normalization(
```

Comment:
Why not self.english_string_normalization()?

Reply:
`normalize_text` is static (`name 'self' is not defined).

Maybe this is a good excuse to decouple the transcript processing functions from the Manifest class? e.g. transcript_tokenize_basic, text_normalization_basic, transcript_tokenize_EN, text_normalization_EN

Reply:
I've been thinking about the onion discussion lately. There really is no need for the Manifest class at all. All it does it text cleaning and tokenization, which could all be helper functions. But I think that's beyond the scope of this PR.

If english_string_normalization is not used anywhere else, let's move it into ManifestEN.normalize_text()

Reply:
why wouldn't text normalization simply be another neural module?  That is much more in keeping with the NM design concept - we are defining a data flow graph of transformations.  Otherwise you are simply creating a separate non-generic framework for a specific data processing step.  That is counter to the overall system design.

Reply:
There is no option nor support for any Neural Modules before a DataLayerNM. A DataLayerNM must be the first module in your call chain in the current design.
Of course you can just return unnormalized text from your dataloader but it is much easier to prune it prior to constructing your dataset than to do it on the fly during training.